File: v1x1-web/app/model/state/configuration_definition_set.ts
import {V1x1ConfigurationDefinition} from "../api/v1x1_configuration_definition";

export class ConfigurationDefinitionSet {
  global: V1x1ConfigurationDefinition;
  user: V1x1ConfigurationDefinition;

  constructor(global: V1x1ConfigurationDefinition, user: V1x1ConfigurationDefinition) {
    this.global = global;
    this.user = user;
  }
}

File: v1x1-modules/v1x1-modules-core/v1x1-modules-core-api/src/main/java/tv/v1x1/modules/core/api/api/rest/ConfigurationDefinitionSet.java
package tv.v1x1.modules.core.api.api.rest;

import com.fasterxml.jackson.annotation.JsonProperty;

public class ConfigurationDefinitionSet {
    @JsonProperty
    private ConfigurationDefinition global;
    @JsonProperty
    private ConfigurationDefinition user;

    public ConfigurationDefinitionSet() {
    }

    public ConfigurationDefinitionSet(final ConfigurationDefinition global, final ConfigurationDefinition user) {
        this.global = global;
        this.user = user;
    }

    public ConfigurationDefinition getGlobal() {
        return global;
    }

    public void setGlobal(final ConfigurationDefinition global) {
        this.global = global;
    }

    public ConfigurationDefinition getUser() {
        return user;
    }

    public void setUser(final ConfigurationDefinition user) {
        this.user = user;
    }
}
